test/cbus_elixir_web/controllers/meeting_registration_controller_test.exs

```diff
@@ -0,0 +1,20 @@
+defmodule CbusElixirWeb.MeetingRegistrationControllerTest do
+  use CbusElixirWeb.ConnCase
+
+  test "GET /", %{conn: conn} do
+    conn = get(conn, meeting_registration_path(conn, :index))
+    assert html_response(conn, 200) =~ "Registration for"
+  end
+
+  test "Shows correct meeting day of", %{conn: conn} do
+    date = "2018-03-25"
+    conn = get(conn, "/?date=#{date}")
+    assert html_response(conn, 200) =~ "2018-04-03"
+  end
+
+  test "Shows correct meeting day of meeting", %{conn: conn} do
+    date = "2018-04-03"
+    conn = get(conn, "/?date=#{date}")
+    assert html_response(conn, 200) =~ "2018-04-03"
```

Comment:
We should check for the title of the meeting rather than just the date is rendered correctly

Reply:
I will re-name this test. This test is testing that the site shows the current meeting on the day of the meeting. 
Could you clarify 'title' a little more, I am making the assumption that you referring to a title of a meeting but I only see 1 field for a meeting of 'date'. Am I misunderstanding your reference to 'title'?

Reply:
You're totally right. There's no other field to present than the date itself.